packages/insomnia-app/app/ui/components/modals/generate-code-modal.tsx

```diff
@@ -13,8 +14,8 @@ import ModalFooter from '../base/modal-footer';
 import ModalHeader from '../base/modal-header';
 import CodeEditor from '../codemirror/code-editor';
 
-const DEFAULT_TARGET = availableTargets().find(t => t.key === 'shell');
-const DEFAULT_CLIENT = DEFAULT_TARGET.clients.find(t => t.key === 'curl');
+const DEFAULT_TARGET = HTTPSnippet.availableTargets().find(t => t.key === 'shell');
```

Comment:
right now, this is typed as `HTTPSnippetTarget | undefined`, but we know it's defined

```suggestion
const DEFAULT_TARGET = HTTPSnippet.availableTargets().find(t => t.key === 'shell') as HTTPSnippetTarget;
```

Reply:
cb6603e0ea8539dbf8e5e5530dcd41d87c9cef5c